src/sentry/web/api.py

```diff
@@ -853,25 +817,10 @@ def post(self, request, project, **kwargs):
             unreal = process_unreal_crash(request.body, request.GET.get(
                 'UserID'), request.GET.get('AppEnvironment'), event)
 
-            refactor_enabled = _minidump_refactor_enabled(project.id)
-            metrics.incr(
-                'symbolicator.minidump-refactor-enabled',
-                tags={'value': refactor_enabled, 'endpoint': 'unreal'}
-            )
-
-            if refactor_enabled:
-                process_state = None
-            else:
-                process_state = unreal.process_minidump()
-
-            if process_state:
-                merge_process_state_event(event, process_state)
-            else:
-                # TODO(markus): Remove after refactor rolled out
-                apple_crash_report = unreal.get_apple_crash_report()
-                if apple_crash_report:
-                    merge_apple_crash_report(apple_crash_report, event)
-                    is_apple_crash_report = True
+            apple_crash_report = unreal.get_apple_crash_report()
+            if apple_crash_report:
+                merge_apple_crash_report(apple_crash_report, event)
+                is_apple_crash_report = True
```

Comment:
This is more a note to self for the full review: This flag should no longer be necessary. Also, we should probably move processing of the Apple crash report fully into `process_event` to avoid overwrites in the potential case that both a minidump and an ACR are present.

Reply:
Right now this is necessary to know if we're dealing with a minidump (`is_apple_crash_report` implies `not is_minidump`). It sucks.